changelogs/7.0.asciidoc

```diff
@@ -3,13 +3,25 @@
 
 https://github.com/elastic/apm-server/compare/6.7...7.0[View commits]
 
+//* <<release-notes-7.0.1>>
 * <<release-notes-7.0.0>>
 * <<release-notes-7.0.0-rc2>>
 * <<release-notes-7.0.0-rc1>>
 * <<release-notes-7.0.0-beta1>>
 * <<release-notes-7.0.0-alpha2>>
 * <<release-notes-7.0.0-alpha1>>
 
+////
+[[release-notes-7.0.1]]
+=== APM Server version 7.0.1
+
+https://github.com/elastic/apm-server/compare/7.0.0...7.0.1[View commits]
```

Comment:
For this link to (eventually) work, it needs a `v` or two:
```suggestion
https://github.com/elastic/apm-server/compare/v7.0.0...v7.0.1[View commits]
```

Reply:
great catch, thank you